### DISCUSSION TITLE
麻烦更新一下文档的Dialog 对话框，visible属性替换了v-model，我搞了2小时才找到错误

### DISCUSSION THREAD

**Discussion by yyyyzwd** [element-plus/element-plus#2173] [Ideas]
麻烦更新一下文档的Dialog 对话框，visible属性替换了v-model，我搞了2小时才找到错误，哭了

**Comment by iamkun** [comment]
will do thanks

**Comment by 0song** [comment]
fixed in https://github.com/element-plus/element-plus/pull/2175
